_commands/basics/id.md

```diff
@@ -16,9 +17,33 @@ ToDo: Needs completion
 
 #### Example command
 
-##### Break it down
+to find a specific user's UID, add -u {username}. Moreover, id -G USERNAME will find all the groups a user belongs to. Force id to display the name of the UID or GID instead of ugly numbers by passing the -n options
+
+~~~ bash
+$id -ng username
+staff eecsstudents club1 club2
+~~~
 
 #### Example command
+To find all the groups associated with a user called exampleuser, enter:
+
+~~~ bash
+$ id exampleuser
+~~~
+
+sample output:
+
+~~~ bash
+uid = 0(exampleuser) gid=0(eecs) groups=1(computing), 1(hello),2(this), 3(is), 4(an), 5(example)
+~~~
 
-##### Break it down
+### Useful options:
```

Comment:
You don't need to make a new pull request. If you push more commits to the
same branch you made the pull request from, it will automatically update.

Sent from a phone so I didn't look if this already just worked for you.
On Sun, Mar 27, 2016 at 20:58 maydent <notifications@github.com> wrote:

> In _commands/basics/id.md
> <https://github.com/c4cs/c4cs.github.io/pull/161#discussion_r57537961>:
>
> >
> > -##### Break it down
> > +### Useful options:
>
> Hi Pat, I added all of that necessary info but now I am having trouble
> making a pull request. The button is unclickable and i can't figure out
> why. do you have any suggestions?
> … <#m_8294638782992121654_>
> On Sun, Mar 27, 2016 at 5:00 PM, Pat Pannuto <notifications@github.com>
> wrote: In _commands/basics/id.md <#161 (comment)
> <https://github.com/c4cs/c4cs.github.io/pull/161#discussion_r57534734>>:
> > > -##### Break it down > +### Useful options: I don't think the whole
> option list is that useful. For example, on my machine $ id -Z id:
> --context (-Z) works only on an SELinux-enabled kernel Not a whole lot of
> flags are that great for this command. Maybe replace this with a breakdown
> of what id prints when there are no options given: $ id uid=1000(ppannuto)
> gid=1000(ppannuto)
> groups=1000(ppannuto),4(adm),24(cdrom),27(sudo),30(dip),46(plugdev),113(lpadmin),128(sambashare),999(vboxsf)
> With no options, `id` tries to print an estimate of all the "useful"
> information for a user. The uid is the user id, a number assigned to each
> user on the machine. The gid is a little more confusing, there's a [good
> summary here](
> http://www.linuxquestions.org/questions/linux-newbie-8/why-is-the-group-name-and-user-name-the-same-913180/).
> The remaining groups listed... You get the idea — You are receiving this
> because you authored the thread. Reply to this email directly or view it on
> GitHub <
> https://github.com/c4cs/c4cs.github.io/pull/161/files/e21f4daf2cd9db1d3aabf6c38fcd78d5c578f67f#r57534734>
>
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/c4cs/c4cs.github.io/pull/161/files/e21f4daf2cd9db1d3aabf6c38fcd78d5c578f67f#r57537961>
>